tutorials/erb_template.md

```diff
@@ -141,14 +141,13 @@ Instead of simple shapes, you can also use a nested loop to generate 100 actors
     </actor>
 
     <%
-        end
```

Comment:
Are you sure this needs to be removed? There are 2 for loops

Reply:
Oops!  Reverted here 1999945cdbfa42e06758f442da50625316f1e9e2